lib/server.js

```diff
@@ -891,13 +891,15 @@ Server.prototype._run = function _run(req, res, route, chain, cb) {
                 return (next());
             }
 
+            handlerName = (chain[i].handlerName ||
```

Comment:
By the way, CI tests failed again.... without any code change I think it's again a timing problem.... don't know how to get it passed